packages/config/src/index.ts

```diff
@@ -107,6 +118,8 @@ export async function processConfig(
 
         const HandlerCtor: MeshHandlerLibrary = handlerLibrary;
 
+        introspectionCache[source.name] = introspectionCache[source.name] || {};
```

Comment:
Forcing a property definition here doesn't feel ideal.
This seems to be done just for the benefit of the handlers, can we make those more resilient and check if their respective source-name has an entry on the initial cache?

I know it will result in small duplication of code, but it's very simple logic and would live in the place where you expect it to be.

Reply:
So you think we should have it while creating `introspectionCache`?